Cargo.toml

```diff
@@ -30,6 +30,10 @@ include = [
 gexiv2-sys = "1.0"
 libc = "0.2"
 num-rational = { version = "0.2", default-features = false }
+glib-sys = { version = "*", optional = true }
```

Comment:
Is it safe to depend on `version = "*"`, or should we limit it to something known to be supported?